frontend/src/utils/TaskUtils.ts

```diff
@@ -237,19 +237,28 @@ export const awaitsUserRatings = <T extends UserInfo | RoadmapUser>(
   roadmap: T extends UserInfo ? number | Roadmap : Roadmap,
 ) => {
   const roadmapId = typeof roadmap === 'number' ? roadmap : roadmap.id;
-  const type = getType(user, roadmapId);
-  if (type === RoleType.Admin || type === RoleType.Business) {
+  const userType = getType(user, roadmapId);
+
+  // Admins and business users represent customers. Separate logic for determining need for ratings.
+  if (userType === RoleType.Admin || userType === RoleType.Business) {
     const customers = isUserInfo(user)
       ? user.representativeFor?.filter(
           (customer) => customer.roadmapId === roadmapId,
         )
       : (roadmap as Roadmap).customers?.filter((customer) =>
           customer.representatives?.some((rep) => rep.id === user.id),
         );
+
+    // If user represents no customers, always return false
+    if (!customers || customers.length === 0) return () => false;
```

Comment:
With an empty array [] the 'filter' below should return false anyway, but for readability I think this is fine as it is